apps/ledger-live-mobile/src/components/StorylyStories/StorylyProvider.tsx

```diff
@@ -29,10 +29,8 @@ const StorylyProvider: React.FC<StorylyProviderProps> = ({ children }) => {
 
   const storylyRef = useRef<Storyly>(null);
 
-  const {
-    // @ts-expect-error TYPINGS
```

Comment:
@mle-gall 
Why do we ignored intentionally this TS error?
Can we forbid it with TS or our Linter? @gre